Rubrik/Public/Sync-RubrikTag.ps1

```diff
@@ -17,7 +17,7 @@ function Sync-RubrikTag
       https://github.com/rubrikinc/PowerShell-Module
 
       .EXAMPLE
-      Sync-RubrikTag -vCenter 'vcenter1.demo' -Category 'Rubrik'
```

Comment:
There shouldn't be a `vCenter` param being passed in the example at all. If you'd like, just purge it and resubmit the PR.